src/main/java/org/zalando/nakadi/filters/LoggingFilter.java

```diff
@@ -76,27 +74,17 @@ private RequestLogInfo(final HttpServletRequest request, final long requestTime)
         private final RequestLogInfo requestLogInfo;
 
         private AsyncRequestListener(final HttpServletRequest request, final HttpServletResponse response,
-                                    final long startTime, final String flowId) {
+                                     final long startTime, final String flowId) {
             this.response = response;
             this.flowId = flowId;
 
             this.requestLogInfo = new RequestLogInfo(request, startTime);
-            ACCESS_LOGGER.info("{} \"{}{}\" \"{}\" \"{}\" {} {}ms \"{}\" \"{}\" {}B",
-                    requestLogInfo.method,
-                    requestLogInfo.path,
-                    requestLogInfo.query,
-                    requestLogInfo.userAgent,
-                    requestLogInfo.user,
-                    HttpStatus.PROCESSING.value(),
-                    0,
-                    requestLogInfo.contentEncoding,
-                    requestLogInfo.acceptEncoding,
-                    requestLogInfo.contentLength);
+            logToAccessLog(this.requestLogInfo, this.response.getStatus(), 0L);
```

Comment:
@v-stepanov isn't it changing the behavior? Before this change we used to log 102 but now it's not clear if the status in the response is going to be 102. I think this is a special log for calls to consumption API where we only know the real response status code afterwards.

Reply:
Oh crap :( I have not noticed that I changed the behavior. Good catch! Will fix it.

Reply:
Fixed https://github.com/zalando/nakadi/pull/1027/commits/dbae4615dc693b826bf4d8605c58161cadf62e51